apps/heft/src/cli/HeftToolsCommandLineParser.ts

```diff
@@ -229,6 +228,10 @@ export class HeftToolsCommandLineParser extends CommandLineParser {
     if (currentCwd !== buildFolder) {
       // Update the CWD to the project's build root. Some tools, like Jest, use process.cwd()
       this._terminal.writeVerboseLine(`CWD is "${currentCwd}". Normalizing to project build folder.`);
+      // If `process.cwd()` and `buildFolder` differ only by casing on Windows, the chdir operation will not fix the casing, which is the entire purpose of the exercise.
```

Comment:
Use `trueCasePathSync` like it's used [here](https://github.com/microsoft/rushstack/blob/master/apps/rush-lib/src/api/RushConfiguration.ts)?

Reply:
This was discussed offline, but unfortunately it can't be used here since, while it will change the path to have the correct drive casing, the subsequent `process.chdir` to the correctly cased path is a no-op since the path is the same (even though it has different casing).